integrationtest/showcase/library/src/main/java/org/hibernate/search/integrationtest/showcase/library/config/SessionFactoryConfig.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.showcase.library.config;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.search.backend.elasticsearch.cfg.ElasticsearchBackendSettings;
+import org.hibernate.search.backend.elasticsearch.cfg.ElasticsearchIndexLifecycleStrategyName;
+import org.hibernate.search.backend.elasticsearch.cfg.ElasticsearchIndexStatus;
+import org.hibernate.search.backend.elasticsearch.impl.ElasticsearchBackendFactory;
+import org.hibernate.search.integrationtest.showcase.library.analysis.LibraryAnalysisConfigurer;
+import org.hibernate.search.integrationtest.showcase.library.model.Account;
+import org.hibernate.search.integrationtest.showcase.library.model.Book;
+import org.hibernate.search.integrationtest.showcase.library.model.BookCopy;
+import org.hibernate.search.integrationtest.showcase.library.model.Borrowal;
+import org.hibernate.search.integrationtest.showcase.library.model.Document;
+import org.hibernate.search.integrationtest.showcase.library.model.DocumentCopy;
+import org.hibernate.search.integrationtest.showcase.library.model.Library;
+import org.hibernate.search.integrationtest.showcase.library.model.Person;
+import org.hibernate.search.integrationtest.showcase.library.model.Video;
+import org.hibernate.search.integrationtest.showcase.library.model.VideoCopy;
+import org.hibernate.search.mapper.orm.cfg.HibernateOrmIndexingStrategyName;
+import org.hibernate.search.mapper.orm.cfg.HibernateOrmMapperSettings;
+import org.hibernate.service.ServiceRegistry;
+import org.hibernate.tool.schema.Action;
+
+public class SessionFactoryConfig {
```

Comment:
While it's a good idea, I really don't see how this is related to this commit in any way.

Reply:
You're right, I will push this change in a different commit.